airflow/utils/python_kubernetes_script.jinja2

```diff
@@ -0,0 +1,44 @@
+{#
```

Comment:
This entire file should be placed in the kubernetes provider.

Reply:
currently this template is loaded using the
`write_python_script` function in python_virtualenv.py, it shared by virtualenv and docker decorators. Thats why the jinja template needs to be in the same folder. 
Ideally we should move this function out of python_virtualenv.

Reply:
This _has_ to be in the kube provider folder, else it won't get released with the provider, and it won't work.

Reply:
Can you move this @subkanthi to the cncf kube providers please?

Reply:
This has still not been resolved